pages.no/common/false.md

```diff
@@ -0,0 +1,7 @@
+# false
+
+> Returnerer en utgangskode på 1.
```

Comment:
The verbs should be in `indicative mood`, I don't speak Norwegian but I should check.

Reply:
Noted, I have applied the changes